pkgs/by-name/ka/kanidm/generic.nix

```diff
@@ -49,7 +49,8 @@ rustPlatform.buildRustPackage rec {
     inherit hash;
   };
 
-  KANIDM_BUILD_PROFILE = "release_nixos_${arch}";
+  KANIDM_BUILD_PROFILE =
+    if stdenv.hostPlatform.isDarwin then "release_freebsd" else "release_nixos_${arch}";
```

Comment:
We customize the release for linux. Looking at the upstream freebsd profile, it seems like it would be worthy of customizing as well. https://github.com/kanidm/kanidm/blob/v1.6.3/libs/profiles/release_freebsd.toml

Could we conditional the profile below instead based on platform, and maybe rename it to nixpkgs?

Reply:
Good call out, thanks! Yes, I _was_ planning to use `kanidmd`, however didn't actually test (just assumed 🤦‍♂️ ). So, I've switched this to `release_nixpkgs_${arch}`, and then made the socket path conditional (since that was the piece that was failing for me). Able to get `kanidmd` running locally successfully now 🙌